src/main/java/com/twentyonec/ItemsLogger/ItemPlayer.java

```diff
@@ -56,13 +57,11 @@ public ItemPlayer(final UUID uuid, final String inv, final String cause, final i
 
 	public void savePlayer() {
 
+		plugin.debugMessage("Attempting to log player data");
 		final String sql = "INSERT INTO itemslogger"
 				+ "(uuid, inventory, cause, loc_x, loc_y, loc_z, experience, date, time) " 
-				+ "VALUES ('" + uuid + "','" + inv + "','" 
-				+ cause + "'," + x + "," + y + "," + z + "," 
-				+ experience + ",?" + ",?" + ");";
-		storage.update(sql, date, time);
-
+				+ "VALUES (?,?,?,?,?,?,?,?,?);";
+		storage.update(sql, uuid.toString(), inv, cause, x, y, z, experience, date, time);
 
 	}
 
```

Comment:
Bruh, move `savePlayer` and `loadInventory` to another class. Literally the 3rd time I've told you to do this.